package-examples/nginx/deployment.yaml

```diff
@@ -28,7 +28,7 @@ spec:
     spec:
       containers:
         - name: nginx
-          image: "nginx:1.14.2"
+          image: "nginx:1.16.1"
```

Comment:
why do you need to update this? Just for testing merge? If so choose something that has lower probability of breaking.

Reply:
it is not breaking... updating image is a better way to test the update. We have been doing that for nginx. Shouldn't be a problem.